apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import ProjectCard from './ProjectCard';
+import i18n from "@cdo/locale";
+
+const styles = {
+  grid: {
+    padding: 10,
+    width: 1000
+  },
+  card: {
+    display: "inline-block",
+    padding: 10
+  },
+  labHeading: {
+    textAlign: "left",
+    fontSize: 24,
+    color: "#5b6770",
+    marginLeft: 10,
+    marginRight: 65,
+    borderBottom: "1px solid #bbbbbb",
+    paddingBottom: 10,
+    paddingTop: 40
+  }
+};
+
+const ProjectCardGrid = React.createClass({
+  propTypes: {
+    projects: React.PropTypes.array,
+    galleryType: React.PropTypes.string
```

Comment:
same question about PropTypes. I think galleryType could also be `PropTypes.onOf(['personal', 'class', 'public'])`?